README.md

```diff
@@ -1,3 +1,5 @@
 # aws-pdf-encrypt-sample
 
-<Insert description on what this does>
\ No newline at end of file
+This is an attempt at generating CI/CD pipelines with the help of AWS SAM for the deployment of a PDFEncrypt Lambda function.

```

Comment:
Not really good enough, how about 

1. How to run? 
2. How to use?
3. What tests are there? 
4. How to install or set up?


These information must be in in the readme - see a detailed example

https://github.com/dwyl/learn-aws-lambda